runtime/config-deployer-service/ballerina/APIClient.bal

```diff
@@ -785,7 +785,11 @@ public class APIClient {
                         return e909022("Provided Type currently not supported for GraphQL APIs.", error("Provided Type currently not supported for GraphQL APIs."));
                     }
                 } else {
-                    model:HTTPRouteRule httpRouteRule = {matches: self.retrieveHTTPMatches(apkConf, operation, organization), backendRefs: self.retrieveGeneratedBackend(apkConf, endpointToUse, endpointType), filters: self.generateFilters(apiArtifact, apkConf, endpointToUse, operation, endpointType, organization)};
+                    model:HTTPRouteRule httpRouteRule = {
+                        matches: self.retrieveHTTPMatches(apkConf, operation, organization),
+                        backendRefs: self.retrieveGeneratedBackend(apkConf, endpointToUse, endpointType),
+                        filters: self.generateFilters(apiArtifact, apkConf, endpointToUse, operation, endpointType, organization)
+                    };
```

Comment:
We can remove the else block here (and in the code lines above) to reduce the indentation depth.

``` ballerina 
if apkConf.'type == API_TYPE_GRAPHQL {
    model:GQLRouteMatch[]|error routeMatches = self.retrieveGQLMatches(apkConf, operation, organization);
    if routeMatches is model:GQLRouteMatch[] && routeMatches.length() > 0 {
        model:GQLRouteRule gqlRouteRule = {matches: routeMatches};
        return gqlRouteRule;
    }
     return e909022("Provided Type currently not supported for GraphQL APIs.", error("Provided Type currently not supported for GraphQL APIs."));
}
model:HTTPRouteRule httpRouteRule = {
      matches: self.retrieveHTTPMatches(apkConf, operation, organization),
      backendRefs: self.retrieveGeneratedBackend(apkConf, endpointToUse, endpointType),
      filters: self.generateFilters(apiArtifact, apkConf, endpointToUse, operation, endpointType, organization)
                    };
return httpRouteRule;
```

Reply:
Since more API types will be added in the future, I instead modified the else block to be an if else block.

```
if apkConf.'type == API_TYPE_GRAPHQL {
    model:GQLRouteMatch[]|error routeMatches = self.retrieveGQLMatches(apkConf, operation, organization);
    if routeMatches is model:GQLRouteMatch[] && routeMatches.length() > 0 {
        model:GQLRouteRule gqlRouteRule = {matches: routeMatches};
        return gqlRouteRule;
    } else {
        return e909022("Provided Type currently not supported for GraphQL APIs.", error("Provided Type currently not supported for GraphQL APIs."));
    }
} else if apkConf.'type == API_TYPE_REST {
    {
        model:HTTPRouteRule httpRouteRule = {
            matches: self.retrieveHTTPMatches(apkConf, operation, organization),
            backendRefs: self.retrieveGeneratedBackend(apkConf, endpointToUse, endpointType),
            filters: self.generateFilters(apiArtifact, apkConf, endpointToUse, operation, endpointType, organization)
        };
        return httpRouteRule;
    }
} else {
    return ();
}
```